docs/src/tut/mkt/index.md

```diff
@@ -0,0 +1,1323 @@
+
+* Line 41 picks the `quantity` of that `product` the `Buyer` wants.
+This is also a random number that is not more than `100`.
+
+* Line 43 displays the `name` of the product that the `Buyer` wants to order.
+
+* Line 45 returns the `decision` (i.e. the `choice` and `quantity`) of the `Buyer`.
+
+That completes the `buyerInteract`. 
+
+
+Finally, the type the code below for `commonInteract`:
+
+```reach
+load: /examples/mkt-2-basic/index.mjs
+range: 49 - 61
+```
+
+All `{!rsh} Participant`s will have access to this code.
+
+* Line 49 defines `commonInteract` as a `{!rsh} function` that takes in `person` as an argument.
+
+* Line 50 defines `showResult` as a `{!rsh} function` that takes in `decision` (returned on line 45) and `sellerInfo` as an argument. 
+
+* Lines 51 to 59 log to the console the details of the `product` that the `Buyer` has decided to purchase.
+The same `product` is also logged to the console as the `product` that the `Seller` has decided to sell.
+
+A more detailed breakdown of the code:
+
+i. `${person === "Seller" ? "sell" : "buy"}` means if the `${person}` is `Seller`, log `sell` else log `buy` to the console.
+
+ii. `${decision.quantity}` is the `quantity` that the `Buyer` wants out of the original quantity of the `product`.
+
+iii. `${decision.quantity > 1 ? sellerInfo.products[decision.choice].units : sellerInfo.products[decision.choice].unit}` is either the `unit` or `units` of the product the `Buyer` chose. 
+This is dependent on the `quantity` that the `Buyer` wants.
+
+iv. `${sellerInfo.products[decision.choice].name}` is the name of the product that the buyer chose.
+
+That seals the frontend for this section.
+Just one more step! 
+
+Go back to the backend file.
+
+
+### {#tut-mkt-basic-connect} Connect Frontend to Backend
+
+This stage is the last one for the basic DApp. 
+It will make the backend to `{!rsh} interact` with the frontend and that in turn, will log outputs in the console.
+To make that happen, all the methods that have just been defined in the frontend, will now be called upon using the `{!rsh} interact` `{!rsh} Object`.
+
+:::note
+Calling `{!rsh} interact` on something in Reach is the same thing as calling upon a `{!rsh} function` to execute the code within it.
+:::
+
+Go back to the backend file. Just below `{!rsh} init();`, type the following code:
+
+```reach
+load: /examples/mkt-2-basic/index.rsh
+range: 50 - 55
+```
+
+This is called a `{!rsh} local step`. 
+
+:::note
+A local step is an action that happens on the `{!rsh} Participant`'s machine. This step is used to execute codes that are only accessible to a particular `{!rsh} Participant`.
+:::
+
+* Line 50 calls the `{!rsh} only()` method on the `Seller`.
+
+* Line 51 has the `Seller` `{!rsh} interact` with the `sellerInfo` `{!rsh} Object`. 
+
+After the `{!rsh} interact`ion is done, the program is then permitted to share whatsoever the result of that `{!rsh} interact`ion is with the public. 
```

Comment:
Can use interaction instead of  `{!rsh} interact`ion, since you are using it as a word and not a reference. Like the difference between "this `function`" and "this functions"